Solutions/CohesitySecurity/Playbooks/Cohesity_Close_Helios_Incident/azuredeploy.json.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
```

Comment:
Can we move this files into Tools/ directory instead? This should be generic script which take the name of the playbook as input and deploys the playbook. I don't think we need one such file for every playbook. Also, as much as possible let's try to seperate development folders and test/script folders.

Please add a description on the top of every new file that we add. We should know what the file is supposed to do by looking at the header.

Reply:
just check, till now, for all of the playbooks we have, they just need one common/same parameter, that's PlaybookName.
so, we could use one sh to serve all.

I will just "hide" this file from common/public-visible branch, for my vrun function of vim, will look for a nearby "azuredeploy.json.sh" file to run, when I press F5 key in azuredeploy.json file.